app/boards/arm/preonic/Kconfig.defconfig

```diff
@@ -0,0 +1,17 @@
+# Preonic keyboard configuration
+
+# Copyright (c) 2022 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+if BOARD_PREONIC_REV3
+
+config ZMK_KEYBOARD_NAME
+	default "Preonic V3"
+
+config ZMK_USB
+	default y
+
+	config ZMK_KSCAN_MATRIX_POLLING
+        default y
```

Comment:
If this is fully required for this to function, it should actually get set in the _defconfig file, which should force it to stay on.

Reply:
`CONFIG_ZMK_KSCAN_MATRIX_POLLING=y` already exists **onpreonic_rev3_defconfig** file, but removing it from **Kconfig.defconfig**